### PR TITLE
Use ‘pkcs' to encrypt

### PR DESCRIPTION
**Pull request by Weber-H2O2**
Now `demo-eigencall` uses public key from `pkcs` to encrypt operands.

### CODE REVIEW

File: l2/eigen-tutorials/packages/demo-eigencall/scripts/exec.js [hunk 1]
```diff
@@ -142,14 +147,51 @@ const sub_cipher_plain = async (contract, cipher, plain) => {
...
+  const cipher = ecies.encrypt(public_key, (new Uint64BE(num)).toBuffer(), options);
+  
+  return cipher
+}
 
 const main = async () => {
-  await eigenLog('Simple eigenCall demo')
+  await arbLog('Simple eigenCall demo')
+  
+  const res = await fetch('http://localhost:3000/store?digest=1', {
```
Comment (eigmax) [inline l2/eigen-tutorials/packages/demo-eigencall/scripts/exec.js:177]: setup the PCKS endpoint like L2RPC by `requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC'])`

File: l2/eigen-tutorials/packages/demo-eigencall/tools/teesdk_util.c [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+int release() {
+  eigen_enclave_info_free(g_enclave_info);
+  eigen_auditor_set_free(g_auditors);
+  return 0;
+}
+
+int main() {
+  const char *pub = "/app/release/services/auditors/godzilla/godzilla.public.der";
```
Comment (eigmax) [inline l2/eigen-tutorials/packages/demo-eigencall/tools/teesdk_util.c:86]: setup the basedir of auditors.   and get the basedir by .env

File: l2/eigen-tutorials/packages/demo-eigencall/tools/teesdk_util.c [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  const char *conf = "/app/release/services/enclave_info.toml";
+  int32_t port = 8082;
+  int result = 0;
+#if 0
+  const char *method = "echo";
+  const char *args = "Hello Eigen";
+  const char *uid = "uid";
+  const char *token = "token";
+#else
+  const char* method = getenv("TEESDK_METHOD");
```
Comment (eigmax) [inline l2/eigen-tutorials/packages/demo-eigencall/tools/teesdk_util.c:97]: This also can be initialized by .env.

**Comment by Weber-H2O2** [comment]
@eigmax ENV related variables will be set in *.env*. I'll improve it.

**Comment by Weber-H2O2** [comment]
The latest commit need to test.

**Review by eigmax — APPROVED** [review]

**Pull request merged**